frontend/src/scenes/settings/environment/SessionsV2JoinModeSettings.tsx

```diff
@@ -0,0 +1,57 @@
+import { useActions, useValues } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonRadio, LemonRadioOption } from 'lib/lemon-ui/LemonRadio'
+import { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+
+import { HogQLQueryModifiers } from '~/queries/schema/schema-general'
+
+type SessionsV2JoinMode = NonNullable<HogQLQueryModifiers['sessionsV2JoinMode']>
+
+const options: LemonRadioOption<SessionsV2JoinMode>[] = [
+    {
+        value: 'string',
+        label: (
+            <>
+                <div>String</div>
+            </>
+        ),
+    },
+    {
+        value: 'uuid',
+        label: (
+            <>
+                <div>UUID</div>
+            </>
+        ),
+    },
+]
+
+export function SessionsV2JoinModeSettings(): JSX.Element {
```

Comment:
This is all hidden behind a FF, I'm expecting that only our team will ever see this (when we dogfood it / see how much difference it made)